pkg/handler/admin/admin.go

```diff
@@ -37,6 +39,37 @@ func (base *Controller) GetUsers(c *gin.Context) {
 
 }
 
+<
+// Delete ALL users
+func (base *Controller) DeleteUsers(c *gin.Context){
+	err := admin.DeleteUsers()
+	if err != nil {
+		rd := utility.BuildErrorResponse(http.StatusInternalServerError, "error", "Unable to delete all users", err,nil)
+		c.JSON(http.StatusInternalServerError,rd)
+		return
+	}
+	rd := utility.BuildSuccessResponse(http.StatusOK, "success", gin.H{"message" : "All users deleted successfully"} )
+	c.JSON(http.StatusOK, rd)
+
+}
+
+
+
+// Delete User
+func (base *Controller) DeleteUser(c *gin.Context){
+	username := c.Param("username") //string
+	err := admin.DeleteUser(username)
+	if err != nil {
+		rd := utility.BuildErrorResponse(http.StatusInternalServerError, "error", "Unable to delete user", err,nil)
+		c.JSON(http.StatusInternalServerError,rd)
+		return
+	}
+	rd := utility.BuildSuccessResponse(http.StatusOK, "success", gin.H{"message" : "user deleted successfully"} )
+	c.JSON(http.StatusOK, rd)
+}
```

Comment:
no code to verify jwt token?
please include that.
also what if the username is not provided?
you're also not checking that.

Reply:
@Scoba said it is not necessary to use token

Reply:
Token validation added